carsus/io/cmfgen/base.py

```diff
@@ -719,6 +794,17 @@ def _get_levels_lines(self, data):
         lines = lines[['energy_lower', 'energy_upper', 
                        'gf', 'j_lower', 'j_upper', 'wavelength']]
 
+        if 'ionization_energy' in reader.keys():
+            ionization_energies = pd.DataFrame.from_records(ioz_list)
+            ionization_energies['ionization_energy'] = (ionization_energies['ionization_energy'].values / u.cm).to('eV', equivalencies=u.spectral()).value
+            ionization_energies = ionization_energies.set_index(['atomic_number', 'ion_charge']).squeeze()
+            self.ionization_energies = ionization_energies
+
+        if 'cross_sections' in reader.keys():
+            cross_sections = pd.concat(pxs_list)
```

Comment:
```suggestion
            cross_sections = pd.concat(pxs_list)
            cross_sections.rename({'ion_charge': 'ion_number', 'label': 'level_number'})
```
To make it compatible with TARDIS.

Reply:
Dropped the commit, this should be done in the output module (see https://github.com/tardis-sn/carsus/issues/214)